packages/react-renderer-demo/src/doc-components/examples-texts/wizard.md

```diff
@@ -39,6 +40,58 @@ another option is to use custom function. The custom function receives as the fi
 nextStep: ({ values }) => (values.aws === '123' &&& values.password === 'secret') ? 'secretStep' : 'genericStep'
 ```
 
+D) **conditional submit step**
+
+Based on a form state value, a step can change into the last step in the wizard and show *submit* button instead of *next* button.
+
+With default constant:
+
+```jsx
+import { CONDITIONAL_SUBMIT_FLAG } from '@data-driven-forms/common/wizard'; // const value = @@ddf-common-wizard__conditional-submit-step
+
+...
+// next step definition
+
+nextStep: {
+        when: 'source-type',
+        stepMapper: {
+          aws: 'aws-step',
+          google: CONDITIONAL_SUBMIT_FLAG,
+          ...
+        },
+},
+```
+
+Or with custom constant value defined by `conditionalSubmitFlag` prop:
+
+```jsx
+const schema = {
+  fields: [{
+    {
+      "component": "wizard",
+      "name": "wizard",
+      // custom flag definition
+      conditionalSubmitFlag: 'make-me-final-step',
```

Comment:
what about to call it: `submitStep`?

And what about to support also array of strings? 🤔

Reply:
Maybe. What would be the use case? I am trying to think why would you need multiple flags for the same purpose. The only I can think of would be using two different wizards in a single form but that in itself does not make a whole lot of sense.

Or having two different flags in two different forms, but at that point, you have a unique flag per form.

Reply:
The question is "why not"? It would bring better flexibility and the code change is just changing `===` to `.includes()`

Reply:
The why not is quite a simple answer for me. Because nobody has asked for it. There is a lot of "why not" features we are not supporting (remember schema parsers?).

Reply:
Yeah, of course. If anyone would need this, we can change it,